default.yaml

```
@@ -1,5 +1,7 @@
 name: "Galoy Banking"
 
+lightningAddressDomain: "bbw.sv"
```

```suggestion
lightningAddressDomain: "pay.bbw.sv"
```

currnetly we use the pay subdomain - the logic is in the galoy-pay server. 

would be great to be at the top level thought, we would a redirection for that

---

Updated